indigo_app/views/works.py

```diff
@@ -659,18 +659,21 @@ def link_amendment(self, info, form):
                         )
 
                 except Amendment.DoesNotExist:
-                    amendment = Amendment()
-                    amendment.amended_work = amended_work
-                    amendment.amending_work = info['work']
-                    amendment.created_by_user = self.request.user
+                    if info.get('commencement_date') or info['work'].commencement_date:
+                        amendment = Amendment()
+                        amendment.amended_work = amended_work
+                        amendment.amending_work = info['work']
+                        amendment.created_by_user = self.request.user
 
-                    if info.get('commencement_date'):
-                        amendment.date = info.get('commencement_date')
+                        if info.get('commencement_date'):
```

Comment:
Perhaps:

`amendment.date = info.get('commencement_date') or info['work'].commencement_date`

Or, stash it as `date` just before the if and re-use it.